scalding-parquet-scrooge/src/main/scala/com/twitter/scalding/parquet/scrooge/ParquetCollectionFormatForwardCompatibility.scala

```diff
@@ -0,0 +1,153 @@
+package com.twitter.scalding.parquet.scrooge
+
+import java.util
+
+import org.apache.parquet.schema.Type.Repetition
+import org.apache.parquet.schema.{GroupType, MessageType, Type}
+import org.apache.parquet.thrift.DecodingSchemaMismatchException
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable
+
+object ParquetCollectionFormatForwardCompatibility {
+
+  private val LOGGER = LoggerFactory.getLogger(getClass)
```

Comment:
Nit: `private[this] val logger = LoggerFactory.getLogger(getClass)` seems to be the convention used elsewhere in the existing code.

Reply:
Addressed.